R/html-tweak.R

```diff
@@ -206,12 +206,18 @@ tweak_homepage_html <- function(html,
   # logo
   if (!is.null(logo) && bs_version > 3) {
     # Remove logo if added to h1
-    xml2::xml_remove(
-      xml2::xml_find_first(
-        html,
-        ".//h1/img[contains(@src, 'logo')]"
+    # Simple img
+    logo_node <- xml2::xml_find_first(html, ".//h1/img[contains(@src, 'logo')]")
```

Comment:
Could you just make it this?

```suggestion
    logo_node <- xml2::xml_find_first(html, ".//h1//img[contains(@src, 'logo')]")
```

But I guess that would leave an empty link which would be confusing

Reply:
Confusing + bad for accessibility